docs/create_topology.md

```diff
@@ -39,7 +39,7 @@ A deployment yaml file specifies 4 things (*optional in italics*):
 4. *A list of controller specs*
 
 A full definition for valid fields in the deployment yaml can be found within
-[deploy/deploy.go](https://github.com/openconfig/kne/blob/816133f1cb563555bcdcb12eb27874b77dd41d1d/deploy/deploy.go#L212).
+[deploy/deploy.go](https://github.com/openconfig/kne/deploy/deploy.go#L212).
```

Comment:
these links dont work for me unless there is /blob/main included

on another note, the links with line numbers were pinned to a specific commit because often times these line numbers quickly become out of date or misleading when changing other unrelated lines in the same file, but at the same time new file changes are not reflected if the section we are referring to actually has meaningful changes. so there is a tradeoff here, up to you and what you feel is best